src/dbnode/storage/types.go

```diff
@@ -352,6 +352,11 @@ type databaseNamespace interface {
 		flush persist.FlushPreparer,
 	) error
 
+	// ColdFlushIndex flushes unflushed in-memory indexes of ColdWrites.
+	ColdFlushIndex(
```

Comment:
Should we rename `FlushIndex` above to `WarmFlushIndex` to keep naming consistent?